packages/playground/src/utils/types.ts

```diff
@@ -21,4 +21,6 @@ export interface ApplicationCard {
   icon: string;
   route: string;
   tags?: string[];
+  isNew: boolean;
+  createdAt: number;
```

Comment:
Please make this optional. This way, you won't need to add it to the old solutions.

Reply:
The createdAt I mean.

Reply:
the old solutions could have releases too